src/Doctrine/Annotation/Tokens.php

```diff
@@ -295,7 +295,7 @@ public function offsetSet($index, $token)
             }
 
             throw new \InvalidArgumentException(sprintf(
-                'Token must be an instance of Symfony\\CS\\Doctrine\\Annotation\\Token, %s given.',
```

Comment:
are there more references to `Symfony\CS` left in the project you think?

Reply:
```
ker@dus:~/github/FriendsOfPHP/PHP-CS-Fixer λ git grep CS\\\\ | grep -v CHANGELOG.md | grep -v UPGRADE.md
tests/Fixtures/FixCommand/TextDiffTestInput.php:14: * Input for @see \Symfony\CS\Tests\Console\TextDiffTest.
tests/Fixtures/Integration/misc/class_definition,no_trailing_whitespace.test:11:    Symfony\CS\Tests\Fixer,
tests/Fixtures/Integration/misc/class_definition,no_trailing_whitespace.test:21:    Symfony\CS\Tests\Fixer,\RFb,
```

Reply:
Ah I only looked in `src`, shall I update those too?

Reply:
please do so ;)